### PR TITLE
add property to item for testing

### PR DESCRIPTION
**Pull request by toban**
Extending the current test-case to adding a property and verifying the results a bit more in detail

### CODE REVIEW

File: test_scripts/selenium/queryservice-ui/queryservice-ui.page.js [hunk 1]
```diff
@@ -17,6 +17,17 @@ class QueryServiceUI extends Page {
...
 
+	resultIncludes( prop, value ) {
+		const text = this.resultTable.getText();
+		if ( !value ) {
+			return text.includes( prop );
+		}
+
+		const regexp = new RegExp( '(' + prop + ')(\\s+)(' + value + ')' );
+		const matches = text.match( regexp );
+		return matches !== null && matches.includes( prop ) && matches.includes( value );
```
Comment (mariushoch) [inline test_scripts/selenium/queryservice-ui/queryservice-ui.page.js:28]: The additional `String.includes` is redundant here: If the regexp matches, both `prop` and `value` are already part of `text`.

**Comment by mariushoch** [comment]
Seems good to go, but giving you the chance to address the nit pick.

File: test_scripts/selenium/queryservice-ui/queryservice-ui.page.js (hunk 1 above)
Comment (toban) [reply test_scripts/selenium/queryservice-ui/queryservice-ui.page.js:28]: It's not a string that comes back from the text.match but an array with all the groups that matched. So the prop could match but the value might not be there.

Comment (mariushoch) [reply test_scripts/selenium/queryservice-ui/queryservice-ui.page.js:28]: You are right about the array part, but still, matches will be null if only part of the string matched.

```
"foo ba".match(new RegExp( '(foo)(\\s+)(bar)' ));
null
"fo bar".match(new RegExp( '(foo)(\\s+)(bar)' ));
null
"foo bar".match(new RegExp( '(foo)(\\s+)(bar)' ));
Array(4) [ "foo bar", "foo", " ", "bar" ]
```

Comment (toban) [reply test_scripts/selenium/queryservice-ui/queryservice-ui.page.js:28]: My bad, should have confirmed that. will remove it!

**Pull request merged**